code/modules/mob/living/carbon/human/status_procs.dm

```diff
@@ -18,12 +18,16 @@
 /mob/living/carbon/human/Unconscious(amount, updating = 1, ignore_canunconscious = 0)
 	amount = dna.species.spec_stun(src,amount)
 	if(has_trait(TRAIT_HEAVY_SLEEPER))
-		amount *= rand(1.25, 1.3)
+		amount *= 1.25
+	if(has_trait(TRAIT_LIGHT_SLEEPER && !TRAIT_HEAVY_SLEEPER))
```

Comment:
we have quirk_blacklist for this

Reply:
Sorry, Im a bit new to coding and the codebase, and this was the easiest and quickest solution I could think of.

Reply:
Fixed!